docs/sources/alerting/unified-alerting/_index.md

```diff
@@ -24,3 +24,29 @@ Before you begin using Grafana alerting, we recommend that you familiarize yours
 
 - The Grafana alerting system can retrieve rules from all available Prometheus, Loki, and Alertmanager data sources. It might not be able to fetch rules from other supported data sources.
 - We aim to support the latest two minor versions of both Prometheus and Alertmanager. We cannot guarantee that older versions will work. As an example, if the current Prometheus version is `2.31.1`, we support >= `2.29.0`.
+
+## Performance Considerations.
+
+Grafana Alerting now supports multi-dimensional alerting, or templatized alerting, so that one alert rule can generate many separate alerts. For example, one alert rule could generate an alert for each VM that's CPU is maxed out. This new capability adds some new performance considerations.
+
+Evaluating Alerting rules consumes RAM and CPU to compute the output of an alerting query, and network resources to write the results to the Grafana SQL database. The SQL database's network connections are often saturated before RAM or CPU are exhausted. The configuration of alert rules affects how many resources the Alertmanager consumes, and therefore the maximum number of rules a given configuration can support.
+
+These load factors include:
+
+1. The **frequency of rule evaluation**, controlled by the "Evaluate Every" field in the alert editor. Use the lowest acceptable evaluation frequency to support more concurrent rules.
+2. The **cardinality of the rule's result set**. For example, you might be monitoring API response errors by monitoring errors for every API path, on every VM in your fleet. This has a cardinality of # paths x # VMs. Where possible, reduce the cardinality of a result set - perhaps by monitoring errors-per-VM instead for each path, for each VM.
+3. The **complexity of the alerting query**. Queries that datasources are able to process and respond to quickly consume fewer resources. This is less important than the frequency of evaluation and the cardinality of the result set, but if you have reduced those as much as you can, looking at the performance of individual queries could make a difference.
+
+Each evaluation of an alert rule generates a set of "alert instances", one for each member of the result set. The state of all alert instances are written to Grafana's SQL database in the `alert_instance` table.
+
+To estimate the load that alerting queries are putting on your system, you can count the number of alert instances that have been updated recently. This is a rough estimate - if you have some rules evaluated every 30m, and others every 10s, load will be unevenly distributed.
```

Comment:
```suggestion
To get a feel for the evaluation rate of alerting queries on your system, Grafana Alerting exposes a metric to observe evaluations. You can use the following query to observe the rate of alert evaluations over 5m windows.

`rate(grafana_alerting_rule_evaluations_total[5m])`

Keep in mind, that if you have rules evaluated every 30m, and others every 10s, the load will be unevenly distributed.
```

How do you feel about suggesting the folks observe `grafana_alerting_rule_evaluations_total` instead? I think this might be a bit more accurate than the below because an alert that was evaluated but returned nothing wouldn't update an instance. 

Adding to that, I've suggested a re-word of the first sentence - at its current state is not possible to evaluate the load of alerting queries unless we add more metrics that measure the output of rules or groups (think of something like # of frames per rule). The best we can give you is an evaluation rate, CPU and memory - from there you can reach your own conclusions.